extmod/webrepl/webrepl.py

```diff
@@ -39,13 +50,20 @@ def accept_conn(listen_sock):
         return
     print("\nWebREPL connection from:", remote_addr)
     client_s = cl
-    websocket_helper.server_handshake(cl)
+    if websslrepl:
+        if hasattr(uos, 'dupterm_notify'):
+            cl.setsockopt(socket.SOL_SOCKET, 20, uos.dupterm_notify)
```

Comment:
What is this dupterm_notify about? (Naive question)

Reply:
My idea was to try to modify `webrepl.py` as little as possible to get SSL working and therefore  increase the least amount of bytes so it does not have an impact in more constrained devices (esp8266).  So yes since this was already commented I decided to not duplicate it.